docs/src/pages/demos/pickers/MaterialUIPickers.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
```

Comment:
Nitpick: Named import doesn't add anything. It's equivalent to `import React from 'react'` and `React.PureComponent`. I say this only for consistency.

Reply:
Do we need a pure component?